amep/plot.py

```diff
@@ -914,6 +914,8 @@ def box(axis: mpl.axes.Axes, box_boundary: np.ndarray, **kwargs) -> None:
     -------
     None.
     '''
+    warnings.simplefilter('always', PendingDeprecationWarning)
```

Comment:
already set in header of python document. please remove here.

Reply:
thanks, I made the change as requested :)

Reply:
thank you :)